client/lib/postal-code/index.jsx

```diff
@@ -85,15 +85,18 @@ export function tryToGuessPostalCodeFormat( postalCode, countryCode ) {
 		return postalCode;
 	}
 
-	if (
-		includes( countryCodeData.length, postalCode.length ) &&
-		postalCode.indexOf( countryCodeData.delimiter ) === -1
-	) {
+	const postalCodeWithoutDelimeters = replace( postalCode, /[\s-]/g, '' );
```

Comment:
....now that I think about it, I don't think this will work as expected 😅 Meaning you can't put these shorthands (`\d`, `\s`) in character classes...
So, I guess the previous stuff was good... :whistling:

Reply:
...apparently, yes, you can.
¯\_(ツ)_/¯